google-client-loader.html

```diff
@@ -139,6 +139,10 @@
 
       ready: function() {
         this._loader = document.createElement('google-js-api');
+          
+        if (!this.shadowRoot) { this.createShadowRoot(); }
```

Comment:
have you run into an issue where `this.shadowRoot` has not been defined?

If so, I recommend using `this.attachShadow({mode:'open})` instead of `createShadowRoot` as that is from the v0 spec and is now deprecated